docs/processes/incident_management.md

```diff
@@ -0,0 +1,204 @@
+# Tailored Process for Managing Critical Production Issues for the 121 Platform
+
+---
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Definitions](#2-definitions)
+3. [Incident Reporting and Classification](#3-incident-reporting-and-classification)
+4. [Incident Response and Firefighting](#4-incident-response-and-firefighting)
+5. [Communication Protocols](#5-communication-protocols)
+6. [Resolution and Recovery](#6-resolution-and-recovery)
+7. [Post-Incident Review](#7-post-incident-review)
+8. [Process Flow Summary](#8-process-flow-summary)
+
+---
+
+## 1. Introduction
+
+This document outlines a tailored process for managing critical production issues for the **121 Platform**. The goal is to establish a clear, efficient, and effective approach to handle incidents from initial reporting through resolution and post-incident analysis. This process is designed to accommodate the unique aspects of our platform, team structure, and existing tools, ensuring minimal disruption to services and continuous improvement over time.
+
+## 2. Definitions
+
+- **Critical Production Issue**: An incident that significantly impacts the core functionalities of the 121 Platform, particularly those affecting the "critical paths" such as payment distribution and recipient registration.
+
+- **Critical Paths**: Key functionalities whose failure would severely impact users or core services. (XXX: Tijs or Piotr should probably have a definition of critical paths somewhere?) For the 121 Platform, these include:
+
+  - Payment distribution processes
+  - Recipient registration and management
+  - Data integrity and availability
```

Comment:
We could also define if the issue is internal or external. Idk if I put the comment in the right place. But would be handy to keep this record as well.

Reply:
I suppose that would fall under the "analysis of the root cause"? I can add that explicitly if you think it's useful.

Reply:
Business loves this stuff!

Reply:
> issue is internal or external

What does "internal" and "external" mean"?

If it means: if the issue is caused by a problem/fault in the 121 Platform or an external system like Twilio or that of an FSP? IMO in many cases that would be figured out (much) later when investigating/analysing the problem. And sometimes it will be in the middle, like a network problem.

Reply:
Yes exactly this